sp800-63a/sec5_proofing.md

```diff
@@ -87,76 +90,78 @@ The goal of identity validation is to confirm and establish a linkage between th
 5. The CSP SHALL require operators to have undergone a training program to detect potential fraud and to properly perform a virtual in-process proofing session.
 6. A CSP MAY have an attendant participate in-person, at the same physical location as the applicant, for the entirety of the enrollment and identity proofing session.
-7. The CSP SHALL employ physical tamper detection and resistance features appropriate for the environment in which it is located. For example, a kiosk located in a restricted area or one where it is monitored by a trusted individual requires less tamper detection than one that is located in a semi-public area such as a retail store.
-8. The CSP SHALL ensure that all communications take place over a mutually authenticated encrypted session.
+7. The CSP SHALL employ physical tamper detection and resistance features appropriate for the environment in which it is located. For example, a kiosk located in a restricted area or one where it is monitored by a trusted individual requires less tamper detection than one that is located in a semi-public area such as the concourse of a shopping mall.
+8. The CSP SHALL ensure that all communications take place over a mutually-authenticated encrypted session.
 
 ### <a name="trustref"></a> 5.3.4. Trusted Referee Requirements
 
-The CSP MAY use trusted referees, such as notaries, legal guardians, medical professionals, conservators, persons with power of attorney, or some other form of certified/trained/approved individuals that can vouch for or act on behalf of the individual in accordance with applicable laws, regulations, or agency policy.  The CSP MAY allow an individual that has successfully completed identity proofing to act as a trusted referee for another individual.  The CSP MAY use a trusted referee for both remote and in-person processes.  
+The CSP MAY use trusted referees, such as notaries, legal guardians, medical professionals, conservators, persons with power of attorney, or some other form of trained and approved or certified individuals that can vouch for or act on behalf of the applicant in accordance with applicable laws, regulations, or agency policy.  The CSP MAY allow an applicant that has successfully completed identity proofing to act as a trusted referee for another applicant.  The CSP MAY use a trusted referee for both remote and in-person processes.  
 
-The CSP SHALL establish written policy and procedures as to how a trusted referee is determined and the lifecycle by which the trusted referee retains his/her status as a valid referee, to include any restrictions, as well as any revocation and suspension requirements. 
+The CSP SHALL establish written policy and procedures as to how a trusted referee is determined and the lifecycle by which the trusted referee retains their status as a valid referee, to include any restrictions, as well as any revocation and suspension requirements. 
```

Comment:
I deleted that MAY requirement as it added no value.  But in this case, it is 100% the agency's call, not ours.  We wrote this to acknowledge this corner case exists, and put it in the hands of the agency.  Rather than being silent on it and fielding calls as to whether or not a referee was ok.